### PR TITLE
56920 First pass of sync 526EZ online form with paper form

### LINKED ISSUE #56920: First pass of 526 sync
## Issue Description
Implement the first list of changes for the online 526 form that doesn't require any research

1. Make phone number field on contact information optional
2. Change user flow:
- Move homeless and terminal illness to vet details chapter
- Remove additional benefits flow/screens from disabilities chapter, except for Individual Employability screens which needs to move to additional info section
- Remove FDC program screen from evidence chapter
- Remove bank account info screens from additional chapter

## Front end tasks
- Ensure the page is styled according to design spec. 
- Ensure the back end data from API is displayed in local and review instance
- Ensure unit tests are available. 
- Ensure your code changes are covered by E2E tests
- Run axe checks using the Chrome or Firefox browser plugin
- Test for color contrast and color blindness issues
- Zoom layouts to 400% at 1280px width
- Test with 1 or 2 screen readers
- Navigate using the keyboard only

## Acceptance Criteria
- [ ] Designer comments on this issue to approve page styling 
- [ ] Unit tests pass
- [ ] E2E tests cover current code and regression
- [ ] End-to-end tests show 0 violations.
- [ ] The data returned matches API response (for given user or scenario)
- [ ] All axe checks pass
- [ ] All color contrast checks pass
- [ ] All zoom testing passes
- [ ] All keyboard checks pass 
- [ ] All screenreader checks pass

## Appendix
<details> 

 <summary>Ensure your code changes are covered by E2E tests (expand)</summary>
 
 - Add E2E tests if none exist for this addition/change. 
 
 - Update existing E2E tests if this code will change functionality. 
 
 - Include axe checks, including hidden content
 
</details>

<details> 

 <summary> Run axe checks using the Chrome or Firefox browser plugin (expand)</summary>
 
- Ensure no heading levels are skipped.

- Ensure all buttons and labeled inputs use semantic HTML elements.

- Ensure all buttons, labeled elements and images are identified using HTML semantic markers or ARIA roles.

- Ensure form fields have clearly defined boundaries or outlines.

- Ensure form fields do not use placeholder text. 

- Ensure form fields have highly visible and specific error states. 

</details>

<details> 

 <summary> Test for color contrast and color blindness issues (expand) </summary>
 - All text has appropriate contrast. 

</details>

<details> 

 <summary> Zoom layouts to 400% at 1280px width (expand)</summary> 
 - Ensure readability and usability are supported when zoomed up to 400% at 1280px browser width
 - Ensure no content gets focused offscreen or is hidden from view.
 
</details>

<details> 

 <summary> Test with 1 or 2 screen readers (expand)</summary> 
 
 - Ensure the page includes a skip navigation link. 
 
 - Ensure all links are properly descriptive. 
 
 - Ensure screen reader users can hear the text equivalent for each image conveying information. 

 - Ensure screen reader users can hear the text equivalent for each image button. 

 - Ensure screen reader users can hear labels and instructions for inputs. 
 
 - Ensure purely decorative images are not announced by the screenreader. 
 
</details>

<details> 

 <summary>Navigate using the keyboard only (expand)</summary> 
 
 - Ensure all links (navigation, text and/or image), form controls and page functions can be reached with the tab key in a logical order. 
 
 - Ensure all links (navigation, text and/or image), form controls and page functions can be triggered with the spacebar, enter key, or arrow keys.
 
 - Ensure all interactive elements can be reached with the tab key in a logical order 
 
 - Ensure all interactive elements can be triggered with the spacebar, enter key, or arrow keys.
 
 - Ensure focus is always visible and appears in logical order.
 
 - Ensure each interactive element has visible focus state which appears in logical order.
 

</details>

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

### PR DESCRIPTION
**Pull request by christinec-fftc**
## Summary
- The first pass of 526 sync is to move and remove pages. In preparation for collaboration cycle staging review, made the following 526 flow changes
   - Move housing situation and terminal illness views from additional information chapter to veteran details chapter
   - Remove Fully Developed Claim view
- Page migration will be handled in a subsequent story
- In order to move pages appropriately, they have been duplicated in the form config. Once staging review is complete and we're ready to launch, the dup's will be removed and the original section id's will be used.
- _(Which team do you work for, does your team own the maintenance of this component?)_
   - Disability Benefits Experience

## Related issue(s)

[#56920](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/56920)

## Testing done

- _Describe what the old behavior was prior to the change_
   - Refer to notes in the summary as to where views were moved or removed from
- _Describe the steps required to verify your changes are working as expected_
   - Manually step through 526 to file a disability claim, filing an original claim and BDD claim. See [notes  ](https://docs.google.com/document/d/1OqmIKHlUKY6N78iYJADW3T_LNgKbjWvN69aKdaXbaJM/edit?usp=sharing) for additional notes to help with testing on what should be validated for each flow.
- _Describe the tests completed and the results_
   - Unit tests and e2e tests were successful
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots
n/a

## What areas of the site does it impact?
form 526EZ

### Quality Assurance & Testing

- [n/a] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [n/a] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [n/a] Documentation has been updated ([link to documentation](#) \*if necessary) --> team prefers to make doc updates after staging review
- [n/a] Screenshot of the developed feature is added
- [n/a] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [n/a] Events are being sent to the appropriate logging solution
- [n/a] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [n/a] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by sethdarragile6 — APPROVED** [review]
Great work! 👍

**Comment by christinec-fftc** [comment]
Met with design this morning and we found one additional view to be removed. Putting this back into draft.

**Comment by christinec-fftc** [comment]
Updated requirements just came in from Design + stakeholders. Moving this back to in progress

File: src/applications/disability-benefits/all-claims/config/form.js
Comment (mchae-nava) [inline src/applications/disability-benefits/all-claims/config/form.js:714]: Curious what the intended behavior for this is outside of prod?

**Review by mchae-nava — COMMENTED** [review]
Couldn't step through, but code LGTM maybe outside of the annoying import warnings. Will step through if no one gets to it before I land

File: src/applications/disability-benefits/all-claims/config/form.js
Comment (SamStuckey) [inline src/applications/disability-benefits/all-claims/config/form.js:757]: would `isProduction()` make this untestable in staging?

**Review by mchae-nava — APPROVED** [review]
Thanks for the clarification! LGTM

File: src/applications/disability-benefits/all-claims/config/form.js
Comment (christinec-fftc) [reply src/applications/disability-benefits/all-claims/config/form.js:757]: Team had a quick huddle to go over this story. Yes, that is the goal to hide this view on staging so that we can go through the collab cycle's staging review before releasing to prod. There is another subsequent story to clean this up once we have approval for launch. We also discussed toggling challenges and collab cycle process

**Review by Mottie — APPROVED** [review]
Looks good to me!

**Pull request merged**
